src/lib/firebase.js

```diff
@@ -16,5 +16,10 @@ var firebaseConfig = {
 };
 
 let fb = firebase.initializeApp(firebaseConfig);
+let db = firebase.firestore();
 
-export { fb };
+export function writeToFirestore(collectionName, options = {}) {
+  db.collection(collectionName).add(options);
+}
+
+export { fb, db };
```

Comment:
I really like how you extracted out `writeToFirestore`...this will definitely come in handy down the line. Well done.